doc/src/modules/matrices/matrices.rst

```diff
@@ -1,8 +1,6 @@
 Matrices (linear algebra)
 =========================
 
-.. module:: sympy.matrices.matrices
-
 Creating Matrices
 -----------------
```

Comment:
All of the content here above the autodoc lines at the bottom of the rst file should be moved into a separate rst file that could be linked to from here (perhaps a guide). The number one benefit of using autosummary is making all the different functions/methods more discoverable so we need it to be clear when you come to this page that it has a list of those functions which is not yet immediately obvious because I still need to scroll through a load of stuff to see that list.

Reply:
I agree. I wanted to minimize the diff so that we can focus on just the specific details about autosummary to make sure everything looks tenable before going all in on it. So far there are some concerns that need addressing, like the one I mentioned above.